Sources/PackageCollections/Providers/JSONPackageCollectionProvider.swift

```diff
@@ -27,13 +29,22 @@ struct JSONPackageCollectionProvider: PackageCollectionProvider {
     private let httpClient: HTTPClient
     private let decoder: JSONDecoder
     private let validator: JSONModel.Validator
+    private let signatureValidator: PackageCollectionSignatureValidator
 
-    init(configuration: Configuration = .init(), httpClient: HTTPClient? = nil, diagnosticsEngine: DiagnosticsEngine? = nil) {
+    init(configuration: Configuration = .init(),
+         httpClient: HTTPClient? = nil,
+         signatureValidator: PackageCollectionSignatureValidator? = nil,
+         diagnosticsEngine: DiagnosticsEngine? = nil) {
```

Comment:
we should probably make this non-optional so you dont need create one in L46 - we want this to come from the outside